Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Dockerfile extending the generic Node image with application files for a
```

Comment:
Do we need copyright notice for Dockerfile?

Reply:
Added